tests/test_schemas.py

```diff
@@ -12,15 +13,40 @@ def test_audio_schema():
 
 
 def test_article_schema():
-    from genericsite.schemas import OGArticle, OGProfile
+    a = OGArticle(
+        title="My First Article",
+        description="An article about the first opengraph object",
+        url="https://example.com/",
+        published_time="2022-06-30T12:00:00Z",
+        author=["https://vince.veselosky.me"],
+        tag=["testpost", "ignoreme"],
+    )
+    print(a)
+    assert '''property="og:type" content="article"''' in str(a)
+    assert '''property="article:tag" content="testpost"''' in str(a)
+    assert '''property="article:tag" content="ignoreme"''' in str(a)
+    assert '''property="article:author" content="https://vince.veselosky.me"''' in str(
+        a
+    )
+    assert "2022-06-30 12:00:00+00:00" in str(a)
```

Comment:
I see in other places you're using `datetime.fromisoformat()`. I'd be tempted to put the ISO formated time string in a var, passing that in for published time, and then using the datetime function to convert it for the test assertions. It seems harder to get things out of sync that way.

Reply:
Part of the goal of the test is to test the round-trip of a datetime string. The OGArticle schema will convert the string to a datetime, then serialize it back to a string when it is stringified. This test ensures data is not lost during the round trip (at least for UTC).